### DISCUSSION TITLE
Arm Ring and Bypass Sensors

### DISCUSSION THREAD

**Discussion by timothymhuang** [dgreif/ring#1073] [Q&A] [answered]
In Homebridge, is there a way to arm Ring in away mode, but bypass all the open doors? I can't figure out how to do this so I have to open the Ring app and do it there every time I need a door open while I arm ring.

**Comment by smockle** [comment, marked as answer]
Yes, check the `"nightModeBypassFor"` docs under [Optional Parameters](https://github.com/dgreif/ring/blob/master/homebridge/README.md#optional-parameters). If you set `"nightModeBypassFor"` to `"all"` in your config, then activate Night mode in the Home app, the Ring app will show the system is armed in Away mode, having bypassed sensors as necessary.
